### PR TITLE
Add installation details to README

### PR DESCRIPTION
**Pull request by aistis-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,133 @@
...
+# app/config/routing.yml
+
+tadcka_admin_job_queue:
+    resource: "@AurejaJobQueueBundle/Resources/config/routing.xml"
+```
+
+**Step 7**. Register cronjob to be executed every minute
+
+```
+* * * * * php app/console aureja:job-queue:run default
```
Comment (tadcka) [inline README.md]: - \* \* \* php app/console aureja:job-queue:run

File: README.md [hunk 2]
```diff
@@ -1 +1,133 @@
...
+    queues:
+        - default
+```
+
+**Step 6**. Import AurejaJobQueue routing files
+
+```
+# app/config/routing.yml
+
+tadcka_admin_job_queue:
```
Comment (tadcka) [inline README.md]: aureja_job_queue

File: README.md [hunk 3]
```diff
@@ -1 +1,133 @@
...
+ * @ORM\Entity
+ * @ORM\Table(name="aureja_job_report")
+ */
+class JobReport extends BaseJobReport
+{
+    /**
+	 * {@inheritdoc}
+	 *
+	 * @ORM\ManyToOne(targetEntity="Acme\YourBundle\Entity\JobConfiguration", inversedBy="reports")
+	 * @ORM\JoinColumn(name="configuration_id", nullable=false, onDelete="CASCADE")
```
Comment (tadcka) [inline README.md]: nullable=false

Comment (aistis-) [inline README.md]: Do you mean it should not be defined as `nullable = false`?

**Comment by aistis-** [comment]
@tadcka I've squashed the commts.

**Comment by tadcka** [comment]
Thanks!

**Pull request merged**